### ISSUE TITLE
Widget with the four age groups

### ISSUE THREAD

**Issue by Ali1** [Ali1/imcsand#15]
Alawi habibi

Can we have a widget in the sidebar with the 4 age groups and a picture of each one. Probably as 2 rows. 

![image](https://cloud.githubusercontent.com/assets/218558/6965212/b3975562-d947-11e4-84d0-0a98d707b627.png)

And then the box that's selected goes to the relevant age page. And if that box becomes highlighted in some way to show which page we are on.

This widget will be instead of the menu buttons for the ages and we can use the same design in various parts of the site.

Thanks alawi

**Comment by Ali1** [comment]
sorry for the bad image, the ages are 0-2, 3-6, 7-15 and 16+ but of course text will always be better than image for this writing.

**Comment by alimoshen** [comment]
Allawi these images, would they be an image or an icon? I think the image would work best am I correct?

**Comment by alimoshen** [comment]
Something like this: 
https://www.circumcision-london.co.uk/test-page/
You can enable this in the sidebar tab, the boxes are all menu items and you would need to give each link a class:
![screen1](https://cloud.githubusercontent.com/assets/4508525/6996647/c9b4c0a0-db8b-11e4-8c6c-1f5d0c35d494.jpg)
You can then assign an image to the class through the theme settings tab;
![screen2](https://cloud.githubusercontent.com/assets/4508525/6996649/d79a7b60-db8b-11e4-91b0-55d0cb5e1439.jpg)

Let me know if this works for you.

**Comment by Ali1** [comment]
Alawi that's great. I would be allowed to choose where in the sidebar it
would go, yes?

Thank you

On Sunday, 5 April 2015, alimoshen <notifications@github.com> wrote:

> Assigned #15 <https://github.com/Ali1/imcsand/issues/15> to @Ali1
> <https://github.com/Ali1>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Ali1/imcsand/issues/15#event-273567467>.
>


-- 
Sent from my iPhone

**Comment by alimoshen** [comment]
Yes that's correct, you could have it above the sidebar menu or below ;)
